ffw/UIRPC.js

```diff
@@ -1477,7 +1477,31 @@ FFW.UI = FFW.RPCObserver.create(
                             "longPressAvailable": true,
                             "upDownAvailable": true,
                             "imageSupported": true
-                        }]
+                        }],
```

Comment:
@Ypostolov please revert changes in `UIRPC.js`
Looks like keyboard capabilities are not included for `UI.GetCapabilities` according to API

Reply:
Fixed in 718d529d03402e216d1b77766a4b819e0a00ebaf
Please take a look.